sdk/core/azure-core/src/exception.cpp

```diff
@@ -3,10 +3,12 @@
 
 #include "azure/core/exception.hpp"
 #include "azure/core/http/http.hpp"
-
+#include "azure/core/http/policies/policy.hpp"
+#include "azure/core/internal/json/json.hpp"
```

Comment:
```suggestion
#include "azure/core/http/policies/policy.hpp"
#include "azure/core/internal/json/json.hpp"

```

Reply:
i'm not sure i follow, what is the change here ? the lines look identical

Reply:
Usually we put an extra new line between `#include ""` and `#include <>`